src/app/_components/DistrictSearch/DistrictSearch.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
```

Comment:
🟢 maybe this file could also be renamed to 'component' or something? to avoid duplications DistrictSearch/DistrictSearch.js

Reply:
Maybe, though what about other folders? I'd keep it as it and then maybe it will be nice to create a quick ticket to update everything.

![image](https://github.com/keenethics/mayak/assets/103147058/0fbf4ba4-a592-4b35-9664-6cb67cdc80a8)

Reply:
both of them are alright in current dev

Reply:
Then I will keep "DistrictSearch" name as is